src/harness/database.py

```diff
@@ -48,6 +48,10 @@
 __PASSWORD = 'password'
 __AUTHORIZATION_STRING = 'Basic ' + base64.b64encode(USERNAME+':'+PASSWORD)
```

Comment:
Line   number 49 will prevent code from executing since USERNAME, PASSWORD  are not defined.          The Defined variables start with __USERNAME & __PASSWORD